applications/AISub.md

```diff
@@ -0,0 +1,256 @@
+|     1. | pallet-member                                                     | `create_membership_plan()`: Create new membership plans (admin only)<br>`purchase_membership_plan()`: Allow users to purchase a membership plan<br>`join_membership_plan_as_supplier()`: Suppliers join a membership plan to offer their models<br>`join_membership_plan_as_machine_supplier()`: Machine suppliers join a membership plan to perform calculations for models and receive pool rewards |
+|     2. | pallet-machine-supplier                                           | `create_machine_service()`: Create and register a machine service for running AI models<br>`penalize_slow_performance()`: Penalize machines for slow performance<br>`penalize_failure_to_run()`: Penalize machines that fail to run models as required |
+|     3. | Vue front end: Initial page and image part AI interaction support | Support image type JSON input and output, can directly input images for editing                                                                                                                                                                                                                                                          |
+|     4. | Vue front end: Initial page and video part AI interaction support | Support video type JSON input and output, can directly input videos or images for editing                                                                                                                                                                                                                                                |
+|     5. | AIStream Protocol: Doc and ts-SDK                                 | Protocol documentation for monitoring AISub usage and serving members, as well as the corresponding Golang version of the SDK and TypeScript version of the SDK.                                                                                                                                                                         |
+|     6. | AI supplier demo: Image part                                      | As an example of text and image type access for AI suppliers and machine suppliers, the language is Golang.                                                                                                                                                                                                                              |
+|     7. | AI supplier demo: Video part                                      | As an example of video type access for AI suppliers and machine suppliers, the language is Golang.                                                                                                                                                                                                                                       |
+
+
+
+When the milestones in the grant are completed, we will open source the entire project.
```

Comment:
This is sadly not in line with our grants program. We do not pay for any milestone deliveries which are closed source.

Reply:
This is indeed a mistake on my part, and I have corrected the relevant sections. We will open source the entire project. Our projects will be open source under the GPL-3.0 license.